_vector-search/ai-search/building-flows.md

```diff
@@ -0,0 +1,443 @@
+---
+layout: default
+title: Configuring AI search types
+parent: Building AI search workflows in OpenSearch Dashboards
+grand_parent: AI search
+nav_order: 10
+---
+
+# Configuring AI search types
+
+This page provides example configurations for different AI search workflow types. Each example shows how to tailor the setup to a specific use case, such as semantic search or hybrid retrieval. To build a workflow from start to finish, follow the steps in [Building AI search workflows in OpenSearch Dashboards]({{site.url}}{{site.baseurl}}/vector-search/ai-search/workflow-builder/), substituting your use case from this page into the appropriate parts of the setup. 
+
```

Comment:
Line 11, last sentence: By "this page", do we mean the linked page or this actual page? If the former, it should be "that page".